app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -709,10 +715,16 @@ struct DeviceDelegateOculusVR::State {
     aHeight = scale * (uint32_t)(vrapi_GetSystemPropertyInt(&java, VRAPI_SYS_PROP_SUGGESTED_EYE_TEXTURE_HEIGHT));
   }
 
-  bool Is6DOF() const {
+  bool IsOculusQuest() const {
     return deviceType >= VRAPI_DEVICE_TYPE_OCULUSQUEST_START && deviceType <= VRAPI_DEVICE_TYPE_OCULUSQUEST_END;
   }
 
+  bool Is6DOF() const {
+    // ovrInputHeadsetCapabilities is unavailable in Oculus mobile SDK,
+    // the current workaround is checking if it is Quest.
+    return IsOculusQuest();
+  }
+
```

Comment:
I find having two different `Is6DOF()` functions confusing. Also is it possible to use a Go 3DoF controller with the Quest?

Reply:
One of `Is6DOF()` is for controllers, and the other one is for headsets. Currently, we only use `ControllerState.Is6DOF()` for getting the capability from controllers. We didn't have `Is6DOF()` requirement for headsets yet, I just prepare it for the future. Ideally, `ovrInputHeadsetCapabilities` should tell us the capability  flag headsets, but it doesn't work right now, and I have checked it is the same with OVR mobile samples.